yb-voyager/cmd/exportSchema.go

```diff
@@ -760,11 +762,11 @@ func applyIndexFileTransformations() (*sqltransformer.IndexFileTransformer, erro
 			log.Infof("skipping error while transforming file %s: %v", indexFilePath, err)
 			errMsg = nil
 		}
-		return nil, errMsg
+		return nil, "", errMsg
 	}
 	log.Infof("Schema modifications are applied to INDEX DDLs and the original file is backed up to %s", backUpFile)
 
-	return indexTransformer, nil
+	return indexTransformer, backUpFile, nil
```

Comment:
can we have the indexTransformer store the backupFile? and so you could call indexTransformer.BackupFile() to get the backupfile?